src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -373,11 +373,17 @@ public void verifyMonocolorAd(String slotName) {
 
   public AdsBaseObject waitForPageLoaded() {
     jsActions.waitForJavaScriptTruthy("document.readyState === 'complete'");
+    return this;
+  }
 
-    String waitForGPTJS = "typeof window.googletag === 'object'";
-    jsActions.waitForJavaScriptTruthy(waitForGPTJS);
+  public AdsBaseObject waitForPageLoaded(boolean withGPT) {
```

Comment:
I second @v3rth, it is more readable when invoking this method, you see what it does and you don't have to figure out what mysterious  `true` or `false` means.